Resources/Public/JavaScript/PageView/PageView.js

```diff
@@ -532,6 +828,7 @@ dlfViewer.prototype.init = function(controlNames) {
 
             // Initiate loading fulltexts
             this.initLoadFulltexts();
+            this.initLoadScores();
```

Comment:
If score is empty, this breaks loading the fulltext.
`error: dlfScoreUtil = undefined`

Reply:
Thanks for reporting this! I've opened a new issue: #1407